go-controller/pkg/node/gateway_iptables.go

```diff
@@ -84,18 +84,6 @@ func getSharedGatewayInitRules(chain string, proto iptables.Protocol) []iptRule
 			args:     []string{"-j", chain},
 			protocol: proto,
 		},
-		{
```

Comment:
Had to go look up the iptables flow diagram again :) I think this is right. Incoming traffic to nodePort will hit nat PREROUTING and DNAT to the clsuter IP which is routable afterwards (only matters for local gw mode). Locally generated packet on the host will hit NAT OUTPUT and get DNAT'ed before routing decision.

@alexanderConstantinescu you could remove the NAT prerouting if shared gw mode, or you could put a TODO so we remember to remove it once we remove local gw mode.

Reply:
Is the reason you're saying to remove NAT prerouting because we have OVS flows on `breth0` that will handle that instead?

Reply:
right